docs/includes/usage-examples/FindOneTest.php

```diff
@@ -31,6 +31,6 @@ public function testFindOne(): void
         // end-find-one
 
         $this->assertInstanceOf(Movie::class, $movie);
-        $this->expectOutputRegex('/^{"_id":"[a-z0-9]{24}","title":"The Shawshank Redemption","directors":\["Frank Darabont","Rob Reiner"\],"id":"[a-z0-9]{24}"}$/');
+        $this->expectOutputRegex('/^{"title":"The Shawshank Redemption","directors":\["Frank Darabont","Rob Reiner"\],"id":"[a-z0-9]{24}"}$/');
```

Comment:
We can't preserve the key/property order [without creating a new object/array](https://luigicavalieri.com/blog/renaming-keys-of-multidimensional-associative-array-while-preserving-elements-order/). This overhead and complexity is not justified here. "id" will always be last.

Reply:
I realize this was in the original pattern, so it's not a change from this PR, but what causes the ObjectId to be printed as a plain string? It doesn't look like this library overrides `Model::toJson()`, so is the ObjectId typing dropped entirely for the model's in-memory representation?

Reply:
By default, `id` is casted into string when `Model::toArray()` is called.